less/index.less

```diff
@@ -53,7 +57,173 @@ table {
     box-sizing: border-box;
 }
 
+// Colors:
+@darkColor: #282828;
+@lightColor: #D3DBDE;
+
+// Media Querys:
+
+@mobile: ~"only screen and (max-width: 500px)";
+@tablet: ~"only screen and (max-width: 800px)";
+
 html, body {
-    height: 100%;
+	height: 100%;
+  font-family: sans-serif;
+  line-height: 1.25;
+}
+
+.container {
+  max-width: 1000px;
+  margin: 0 auto;
+
+  @media only screen {
+    background: @lightColor;
+  }
+}
+
+h1 {
+  font-size: 2.4rem;
+}
+
+h2 {
+  font-size: 2rem;
+  border-bottom: 1px solid rgba(0,0,0, 0.7);
+}
+
+h3 {
+	font-size: 1.4rem;
+}
+
+h1, h2, h3 {
+  margin-bottom: 1rem;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.flex(@justify, @align) {
+  display: flex;
+  justify-content: @justify;
+  align-items: @align;
+}
+
+.flipFlex(){
+  flex-direction: column;
+  align-items: center;
+}
+
+header {
+  .banner {
+    .flex(space-between, center);
+    background: @darkColor;
+    color: @lightColor;
+    padding: 1rem 2rem;
+
+    @media @tablet {
+      .flipFlex();
+    }
+  }
+}
+
+nav {
+
+  @media only print {
+    display: none;
+  }
+
+  a{
+    margin-left: 2rem;
+
+    &:first-child {
+      margin-left: 0;
```

Comment:
instead of margins on your a tags for spacing I would refactor your nav to have a width of 50% and maybe around 80% in a media query down the line. then I'd display: flex; and justify-content: space-around (or whichever spacing you like). should look basically the same but you won't have the little crowding issue in the smallest window size